src/universal/components/LinkNewTab/LinkNewTab.js

```diff
@@ -0,0 +1,28 @@
+import React, {PropTypes} from 'react';
+
+const LinkNewTab = (props) => {
+  const { children, href, title } = props;
+  let url = href;
+  // Add http protocol to url if no protocol was provided
+  const urlProtocolRegex = new RegExp(/http:\/\/|https:\/\/|mailto:|ftp:\/\//);
```

Comment:
if a protocol is not included in the href, then it will append the href to the current url. For example: ```[link](google.com)``` will open a new tab to ```http://localhost:3000/team/google.com```

whereas ```[link](http://google.com)``` will open a new tab to google